addOns/exim/src/main/java/org/zaproxy/addon/exim/har/HarImporter.java

```diff
@@ -191,7 +190,7 @@ private void updateProgress(int count, String line) {
         }
     }
 
-    private void completed() {
+    void completed() {
```

Comment:
Should not require this change.

Reply:
Hmmm I rolled one of those back, guess I missed one. Tks!